app/views/admin/questions/_question.html.slim

```diff
@@ -1,20 +1,21 @@
 -@questions.each do |question|
   tr.info
     td
-      =link_to admin_question_path(question)
+      = link_to admin_question_path(question)
         = question.id
     td
-      =link_to admin_topic_path(question.topic_id)
+      = link_to admin_topic_path(question.topic_id)
```

Comment:
I tried this way too. In this case when _level is rendered in question/show it is nowhere to get this 'question' variable.